skills/repo-gardener/scripts/config_check.py

```
@@ -361,6 +385,119 @@ def normalize_tracker(value: Any) -> dict[str, str]:
+        return uses_split_string or bool(
+            wrapped_command and command_uses_string_wrapper(wrapped_command)
+        )
+    # Reserve wrapper switches anywhere in interpreter argv rather than
+    # duplicating each interpreter's evolving option grammar here.
+    return any(
+        option_uses_string_wrapper(executable, option)
+        for option in arguments
+    )
+
+
+def normalize_audit_commands(value: Any, label: str) -> list[list[str]]:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)